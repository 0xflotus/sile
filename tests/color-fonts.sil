\begin[class=book,papersize=129mm x 198mm]{document}
\script[src=packages/color-fonts]
\font[filename=TestCLR-Regular.ttf]
a☺a

a☺🏻a 

ع☺🏻ع 
\end{document}
