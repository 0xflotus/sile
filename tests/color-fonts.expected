<tests/color-fonts.sil>
Set paper size 	365.6692953	561.2598486
Begin page
Mx 	50.35055
My 	73.10614
Set font 	;10;200;regular;normal;;LTR;tests/TestCLR-Regular.ttf
T	3	(a)
Push color	1	0.87843137254902	0.54117647058824
Mx 	60.35055
T	7	()
Pop color
Push color	0.89803921568627	0.6078431372549	0.14509803921569
T	8	()
Pop color
Push color	0.51764705882353	0.21960784313725	0.050980392156863
T	9	(‚ò∫)
Pop color
Mx 	70.35055
T	3	(a)
Mx 	50.35055
My 	88.10614
T	3	(a)
Push color	0.98039215686275	0.87843137254902	0.81176470588235
Mx 	60.35055
T	7	()
Pop color
Push color	0.89803921568627	0.6078431372549	0.14509803921569
T	8	()
Pop color
Push color	0.51764705882353	0.21960784313725	0.050980392156863
T	9	(‚ò∫üèª)
Pop color
Mx 	70.35055
T	3	(a)
Mx 	50.35055
My 	103.10614
Set font 	;10;200;regular;normal;;RTL;tests/TestCLR-Regular.ttf
T	4	(ÿπ)
Push color	0.98039215686275	0.87843137254902	0.81176470588235
Mx 	60.35055
Set font 	;10;200;regular;normal;;LTR;tests/TestCLR-Regular.ttf
T	7	()
Pop color
Push color	0.89803921568627	0.6078431372549	0.14509803921569
T	8	()
Pop color
Push color	0.51764705882353	0.21960784313725	0.050980392156863
T	9	(‚ò∫üèª)
Pop color
Mx 	70.35055
Set font 	;10;200;regular;normal;;RTL;tests/TestCLR-Regular.ttf
T	4	(ÿπ)
[1] Mx 	170.06688
My 	490.44635
Set font 	Gentium;10;200;regular;normal;;LTR
T	17	(1)
End page
Finish

